package.json

```diff
@@ -123,8 +121,7 @@
     "remark-extract-anchors": "1.1.1",
     "remark-frontmatter": "^3.0.0",
     "remark-gfm": "^1.0.0",
-    "remark-html": "^13.0.1",
-    "remark-loader": "^4.0.0",
+    "remark-html": "^13.0.0",
```

Comment:
Any issue after removing it?

Reply:
Nope. We have some remark plugins using it for testing, see https://github.com/webpack/webpack.js.org/blob/0f40d4ed63077ec53426bd349f432fe60225a6a9/src/remark-plugins/remark-custom-asides/index.test.js#L10. That's why I added it back.